pipeline/metadata/test_flatten_base.py

```diff
@@ -1,6 +1,7 @@
 """Unit tests for common flattening functions"""
```

Comment:
I would add tests for [this cert](https://github.com/censoredplanet/censoredplanet-analysis/blob/ba07ca757c241cbbf9895c8d958a5cea1dc881e7/pipeline/metadata/test_flatten.py#L16) which is missing most fields, 

and the self-signed cert @eltsai was having trouble parsing. (Even if it fails we should have the behavior documented)

Reply:
Added a test for one of the self-signed certs.

Reply:
Added a test for the cert with missing fields.